### ISSUE TITLE
Content between comments acts like it's commented out

### ISSUE THREAD

**Issue by josephcsible** [VirusTotal/yara#1011]
Consider this rule:
```
rule MyRule
{
    strings:
        $a = { 41 /* A */ 42 /* B */ }

    condition:
        any of them

}
```
That is clearly supposed to match any file with the hex sequence `41 42`, but it in fact matches any file with just `41`.

**Comment by josephcsible** [comment]
fe7d09e does not seem to have fixed this, and it re-broke #1003.

**Comment by plusvic** [comment]
Oh, you're right. I assumed that #1005 was the cause of this issue because it was very related, but it is not. I've added a test case and will look into it.
